packages/react-tooltip-trigger/src/common/isConformant.ts

```diff
@@ -0,0 +1,10 @@
+import { isConformant as baseIsConformant, IsConformantOptions } from '@fluentui/react-conformance';
+
+export function isConformant<TProps>(testInfo: Omit<IsConformantOptions<TProps>, 'componentPath'>) {
+  const defaultOptions = {
+    asPropHandlesRef: true,
+    componentPath: module!.parent!.filename.replace('.test', ''),
+  };
+
+  baseIsConformant<TProps>(defaultOptions, testInfo);
```

Comment:
Can you change this as follows to match the format from my open PR #17577?
```suggestion
export function isConformant<TProps = {}>(
  testInfo: Omit<IsConformantOptions<TProps>, 'componentPath'> & { componentPath?: string },
) {
  const defaultOptions: Partial<IsConformantOptions<TProps>> = {
    asPropHandlesRef: true,
    componentPath: module!.parent!.filename.replace('.test', ''),
  };

  baseIsConformant(defaultOptions, testInfo);
```